services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.service.ts

```diff
@@ -291,6 +283,50 @@ export class IntersolveVisaService
     return returnData;
   }
 
+  public async getRetrievedAndUpdatedWallet(
```

Comment:
Did mention you want to avoid the update terminology? I think it also better to avoid past tense here as this function also updates and retrieves a wallet

Reply:
Change to RetrieveAndUpdateWallet()